config.xml

```diff
@@ -14,13 +14,13 @@
   <directive action="sum" add="[if?:input_registration==Student Preregistration - $0.00:0:0]" total="total"/>
   <!--Nelnet - update is_test before launch-->
   <!--directive special="nelnet" is_test="N" amount="[total:total]" order_type="CONF27"/>-->
-  <directive special="nelnet" amount="[total:total]" order_type="CONF27"/>
+  <directive special="nelnet" is_test="Y" amount="[total:total]" order_type="CONF27"/>
```

Comment:
The `is_test` parameter is no longer valid; to test point to www-test.bu.edu/phpbin/marconi/ (and we'll deploy changes to the application's test branch), whereas production is www.bu.edu/phpbin/marconi/